pyvista/plotting/tools.py

```diff
@@ -485,7 +530,7 @@ def parse_color(color, opacity=None, default_color=None):
         color='#FFFFFF'""")
     if opacity is not None and isinstance(opacity, (float, int)):
         color = [color[0], color[1], color[2], opacity]
```

Comment:
If we claim that this returns a tuple, we should return a tuple. And numpy dtypes are still excluded here, which is a problem for numpy array valued input.

Reply:
I'm working on that right now, will push part 2 once I'm ready :)
A tuple is already returned right?

Reply:
Ah, sorry, I misread, I thought we were returning this `color` list. Yes, ignore the tuple part then; sorry!